sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumberOperationStatus.cs

```diff
@@ -6,6 +6,7 @@
 namespace Azure.Communication.PhoneNumbers
 {
     [CodeGenModel("PhoneNumberOperationStatus")]
+    [CodeGenSuppress("PhoneNumberOperationStatus")]
```

Comment:
Is there a difference between OperationStatus and OperationType? 
Status is internal readonly and has the CodeGenSuppress attribute while Type was updated to be public and does not have the codegensuppress attribute.